README.md

```diff
@@ -43,13 +42,66 @@ Some obvious cornerstones that must be upheld by all projects are:
+
+#### dApp Support Tiers
+
+The support tier of a project defines the urgency and diligence that PoToC takes when maintaining them. They are derived from the perceived importance for Polkadot as a whole.
+
+Defining these tiers in a direct way is very difficult. Hence, we use the approach of estimating the potential negative impact on Polkadot, if we were to misjudge the tier of a project. Note that in the following definitions, terms like "end-user experience" are used without proper definition.  
+It may sound contradictory to talk about end-users, while PoToC does not aim at end-user software. However, it should hopefully give us a way to categorize projects.
+
+##### Tier D1 - Critical
+
+Software in this tier has the potential to disrupt end-user experience for a large number of users, if it were to malfunction.
+
+Remedy: Must have at least one maintainer at all time who can resolve incidents within 24hrs.
+
+##### Tier D2 - Relevant
+
+Software in this tier has the potential to disrupt end-user experience for a small number of users, if it were to malfunction.
+
+Remedy: Must have at least one maintainer who can resolve incidents within 7 days.
 
-Generally, all tools that are used by The Fellowship to craft, test, and validate updates fall into this category. But only the tools that are predominantly developed for this cause shall be maintained by PoToC. For example, the Rust language is not predominantly developed for Polkadot; hence, it does not fit here.
+##### Tier D3 - Optional
 
-### dApp Tooling
+Software in this tier does not have the potential to negatively affect end-user experience for even a small amount of Polkadot's end-users, if it were to malfunction.
 
-This category contains all tools and libraries that are used to build *on* Polkadot and are predominantly developed for this cause.
+Remedy: None. There are probably alternatives or manual work to get around the malfunction.
+
+PoToC still maintains these projects to foster their future adoption. Projects should aim to soar to a higher tier within at most a year of entering `D3`.
+
+### Fellowship Developer Tooling
+
+Generally, all tools that are used by The Fellowship to craft, test, and validate updates fall into this category. But only the tools that are predominantly developed for this cause shall be maintained by PoToC.
+
+The consistence of this category is mostly circumstantial - through the fact that the Fellowship may use any tools they see fit. Nonetheless, for now it seems like a reasonable approach, since The Fellowship should not have to creep its mission into tooling maintenance. In the future, there may be sub-collectives attached to The Fellowship that assume this work.
+
+#### Fellowship Support Tiers
+
+For Fellowship tooling, only two support tiers exist.
+
+##### Tier F1 - Critical
+
+Software in this tier has the potential to delay the rollout of runtime upgrades or node releases, if it were to malfunction.
```

Comment:
I think that this makes the "tooling" very specific to rust tools. 
E.g. plenty of times in the past there is a runtime upgrade that happens and then tools "follow". 
I never remember some tool, delaying a runtime upgrade